helix-view/src/view.rs

```diff
@@ -314,6 +314,13 @@ impl View {
         self.offset_coords_to_in_view(doc, scrolloff).is_none()
     }
 
+    /// Whether to change a view alignment of current file. You probably don't want to
+    /// change the alignment of current file when open new files in the background (Load).
+    /// Except when the new file is the old file
+    pub fn change_align_view(action: Action, old_id: DocumentId, new_id: DocumentId) -> bool {
```

Comment:
> Note that I would also accept the view as an argument (see signsture above). The view contains the currently active doc_id anyway and that way it's clearer from the signature what's happening. It also removes some extra code around storing the old document I'd that I don't really think is necessary.

Yep, good catch. `view` keeps the current active document, so if we don't open as `Load`, it will have the new doc_id (not the old one), but it doesn't matter because `align_view` will return `true` because `action != Load`. If we do `Load`, the view will still contain the current document id, but for our task it will be the "old" document. Beautiful.